test/fuzzing/fuzzing.go

```diff
@@ -0,0 +1,41 @@
+package fuzz
```

Comment:
Better to add the copyright statement here. 🙏

Reply:
Done! Thank you!